_im-plugin/ism/policies.md

```diff
@@ -162,6 +163,46 @@ Parameter | Description | Type | Required
 
 For information about setting replicas, see [Primary and replica shards]({{site.url}}{{site.baseurl}}/opensearch#primary-and-replica-shards).
 
+### shrink
+
+Allows you to reduce the number of primary shards in your indexes. With this action, you can specify:
+
+- the number of primary shards that the target index should contain.
+- A max shard size for the primary shards in the target index.
+- Specify a percentage to shrink the number of primary shards in the target index.
+
+```json
+"shrink": {
+    "num_new_shards": {
+        "type": "integer"
+    },
+    "max_shard_size": {
+        "type": "keyword"
+    },
+    "percentage_of_source_shards": {
+        "type": "double"
+    },
+    "target_index_name_template": {
+        "source": "the index is {{index-name}}"
+    },
+    "aliases": {
+        "type": "object"
+    },
+    "force_unsafe": {
+        "type": "boolean"
+    }
+}
+```
+
+Parameter | Description | Type | Example | Required
+:--- | :--- |:--- |:--- |
+`num_new_shards` | The maximum number of primary shards in the shrunken index. | integer | `5` | Yes, however cannot be used with `max_shard_size` or `percentage_of_source_shards`
+`max_shard_size` | The maximum size in bytes of a shard for the target index. | keyword | `5gb` | Yes, however cannot be used with `num_new_shards` or `percentage_of_source_shards`
+`percentage_of_source_shards` | Percentage of the number of original primary shards to shrink. This parameter indicates the minimum percentage to use when shrinking the number of primary shards. Must be between 0.0 and 1.0 exclusive.  | Percentage | `0.5` | Yes, however cannot be used with `max_shard_size` or `num_new_shards`
+`target_index_suffix` | Suffix that will be appended to the target index to by shrunk. | text | `shrunken` | No
```

Comment:
This parameter needs to be updated as above, and the default is {{ctx.index}}_shrunken, sorry I don't think I actually specified that before